webapi/src/test/scala/org/knora/webapi/e2e/v2/OntologyV2R2RSpec.scala

```diff
@@ -2810,7 +2819,12 @@ class OntologyV2R2RSpec extends R2RSpec {
     }
 
     // Create a text property.
-
+    val label1 = LangString.make(LanguageCode.en, "blue test text property").fold(e => throw e.head, v => v)
+    val comment1 = Some(
```

Comment:
Could you actually implement value objects for `PropertyLabel` and `PropertyComment`?

Reply:
maybe I make this a follow-up task?